hangman/hangman.py

```diff
@@ -1,7 +1,10 @@
 import random
 def get_target_word():
-    """ This function picks a random string from a list"""
-    my_list = ["We", "eat", "chicken", "and", "turkey"]
+    """Selects a random string from a file"""
+    my_list = []
+    with open("hangman/data/words.txt", "r") as words:
```

Comment:
Isn't hangman the name of this project folder?
The text file should be inside the project folder.

I may be mistaken sha

Reply:
The question says 
Create a folder named data in the root hangman directory.
Download and copy the file https://github.com/Xethron/Hangman/blob/master/words.txt into that folder

Reply:
Yeah, the text file should be inside the `data` folder, the source file should be side by side with the data folder abi?
And both are inside the `hangman` folder.

Reply:
Me sef I've confuse, don't change it oh. @yudori come and check it

Reply:
Yes, thats how it is

Reply:
I think two of you are saying the same thing. @ebosetalee  Using `data/words.txt` doesn't work on your vscode because you didn't add the `hangman/` directory to your PYTHONPATH configuration. 
`hangman/data/words.txt` still references the same file, you're just starting from the root directory as base. Either is fine